code/analyze/ac/plots/de_PlotFFTs.m

```diff
@@ -309,18 +309,30 @@
           pwrLH = squeeze(ffts.orig2D(1,1,:,:)).*(pd<0).*(ffts.pals.an2D<=0.05);
 
           imgRH = ifft2(pwrRH); imgRH=imgRH(1:mSets.nInput(1), 1:mSets.nInput(2));
+          rMin = min(min(imgRH));
```

Comment:
I find `min(imgRH(:))` is a bit clearer, as it says: get the min over all dimensions. Your idiom reads: get the min over the first two dimensions; I have to know that there are only two.